packages/bitstring/bitstring.3.0.0/opam

```diff
@@ -0,0 +1,30 @@
+authors      : [ "Richard W.M. Jones" "Xavier R. Guérin" ]
+bug-reports  : "https://github.com/xguerin/bitstring/issues"
+dev-repo     : "https://github.com/xguerin/bitstring.git"
+doc          : "https://xguer.in/bitstring"
+homepage     : "https://github.com/xguerin/bitstring"
+license      : "LGPLv2+ with exceptions and GPLv2+"
+maintainer   : "Xavier R. Guérin <ghub@applepine.org>"
+opam-version : "1.2"
+version      : "3.0.0"
+
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+
+build-test: [
+  ["jbuilder" "runtest" "-p" name "-j" jobs]
+]
+
+depends: [
+  "jbuilder"                { build              }
+  "ppx_tools_versioned"     { build              }
+  "ocaml-migrate-parsetree" { build & >= "1.0.5" }
+  "ounit"                   { build              }
```

Comment:
This is used only in the tests, why it is flagged as `build` requirement?

Reply:
Only oUnit is used in the tests. The rest is used by the builder and the PPX extension.

Reply:
My point is that I think you only need the test flag:
```
  "ounit"                   { test }
```